collector/src/main/java/io/prometheus/jmx/JmxCollector.java

```diff
@@ -335,9 +337,50 @@ private static boolean isLegalCharacter(char input) {
             (input >= '0' && input <= '9'));
   }
 
-    class Receiver implements JmxScraper.MBeanReceiver {
+    /**
+     * A sample is uniquely identified by its name, labelNames and labelValues
+     */
+    static class SampleKey {
```

Comment:
Should be `private static class SampleKey`.

Reply:
good call.